rewrite.go

```diff
@@ -49,7 +50,12 @@ func rewriteTree(path, qual string, paths []string) error {
 			log.Println("rewrite:", w.Err())
 			continue
 		}
-		if !w.Stat().IsDir() && strings.HasSuffix(w.Path(), ".go") {
+		s := w.Stat()
```

Comment:
For me anyway this makes the intent clearer:

```go
s := w.Stat()
switch s.IsDir() {
  case true:
    if s.Name() == "testdata" {
      w.SkipDir()
      continue
    }
  case false:
    if strings.HasSuffix(w.Path(), ".go") {
      if err := rewriteGoFile(w.Path(), qual, paths); err != nil {
        return err
      }
    }
}
```

Feel free not to change it though.